mycroft/skills/personal/dialog/en-us/who.am.i.dialog

```diff
@@ -0,0 +1,2 @@
+I'm Mycroft. Didn't you know that?
```

Comment:
Does this sound a bit rude? hahah... What about:
"My name is Mycroft and I'm an intelligent personal assistant"